docs/changelog.rst

```diff
@@ -5,6 +5,9 @@ CHANGELOG
 2.100.2+dev (XXXX-XX-XX)
 ------------------------
 
+**Maintenance**
+
+- Upgrade `django-mapentity` to 8.6.0. New authentication system for screamshotter and convertit by token instead of IP detection.
```

Comment:
```suggestion
- Upgrade `django-mapentity` to 8.6.1. New authentication system for screamshotter and convertit by token instead of IP detection.
```